### PR TITLE
Update introduction_guide.rst

### PR DESCRIPTION
**Pull request by yucongo**
## What does this PR do?

Fixes # (issue)

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/introduction_guide.rst [hunk 1]
```diff
@@ -106,7 +106,7 @@ equivalent to a pure PyTorch Module except it has added functionality. However,
     net = LitMNIST()
     x = torch.randn(1, 1, 28, 28)
     out = net(x)
-
+    out.size()
```
Comment (rohitgr7) [inline docs/source/introduction_guide.rst:109]: @yucongo why is this required here?

Comment (Borda) [reply docs/source/introduction_guide.rst:109]: I guess some leftover from debugging...
